iidm/iidm-xml-converter/src/main/resources/xsd/iidm_V1_1.xsd

```diff
@@ -127,11 +128,14 @@
         <xs:complexContent>
             <xs:extension base="iidm:Identifiable">
                 <xs:attribute name="node" use="required" type="xs:int"/>
-                <xs:attribute name="v" use="optional" type="xs:double"/>
-                <xs:attribute name="angle" use="optional" type="xs:double"/>
             </xs:extension>
         </xs:complexContent>
     </xs:complexType>
+    <xs:complexType name="CalculatedBus">
+        <xs:attribute name="nodes" use="required" type="xs:string"/>
+        <xs:attribute name="v" use="optional" type="xs:double"/>
+        <xs:attribute name="angle" use="optional" type="xs:double"/>
```

Comment:
If we agree with @mathbagu 's proposition regarding calculated bus, v and angle will be required